uk.co.compucorp.civicrm.hrcore/backstop_data/scenarios/ssp-la-manager-leave-requests.json

```diff
@@ -1,18 +1,9 @@
 {
-  "scenarios": [{
```

Comment:
Why dod you remove these 2 scenarios, we still need to screenshot of the page without the Modal open